app/models/cinema_hall.rb

```diff
@@ -0,0 +1,2 @@
+class CinemaHall < ApplicationRecord
```

Comment:
`# frozen_string_literal: true` is missing :) Have you run `rubocop -A` before pushing?